Gordon360/Controllers/RecIM/ActivitiesController.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class ActivitiesController : GordonControllerBase
+    {
+        private readonly IActivityService _activityService;
+
+        public ActivitiesController(IActivityService activityService)
+        {
+            _activityService = activityService;
+        }
+
+        ///<summary>Gets a list of all Activities by parameter </summary>
+        ///<param name="active"> Optional active parameter </param>
+        ///<param name="time"> Optional time parameter </param>
+        /// <returns>
+        /// All Existing Activities 
+        /// </returns>
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<ActivityViewModel>> GetActivities([FromQuery] DateTime? time, bool active)
+        {   
+            if (time is null && active)
+            {
+                var activeResults = _activityService.GetActivities();
+                return Ok(activeResults);
+                
+            }
+            var result = _activityService.GetActivitiesByTime(time);
```

Comment:
However you rename the `time` parameter, please do the same with this method on the ActivityService

Reply:
I think it's time we looked and tried to re-think our use case. I don't think we've ever used time and not sure if we ever will unless we implement a filter feature for activity search. @cpabbot  @silasw2  @Kevin-Chu58  thoughts?

Reply:
This filter feature will most likely be an admin thing thus should be relatively non blocking. I personally have the argument to just not have it, and replace it with a `deleted` tag, such that the admin page can have the option of loading in deleted objects to be sorted by the UI. But I don't know the performance implications of this.